reference/library/src/morpheus-master/sass/base/_icons.scss

```diff
@@ -82,13 +82,13 @@
 	.icon-sakai-web-168{					@extend .fa-globe;}
 	.icon-sakai-pasystem{					@extend .fa-bell;}
 	.icon-sakai-feedback{					@extend .fa-paper-plane-o;}
-	.icon-sakai-subsite{				@extend .fa-level-up; }
+	.icon-sakai-subsite{					@extend .fa-level-up;}
 	.icon-sakai-subsites-list{				@extend .fa-level-down; }
-	.icon-sakai-home{                       @extend .fa-university;}
-	.icon-sakai-delete{                     @extend .fa-times;}
-	.icon-sakai-pdf{                        @extend .fa-file-pdf-o;}
-	.icon-sakai-clip{                       @extend .fa-paperclip;}
-	.icon-sakai-check{                      @extend .fa-check;}
-	.icon-sakai-rsf-evaluation{             @extend .fa-commenting;}
-	.icon-sakai-bbb{             @extend .fa-video-camera;}
+	.icon-sakai-home{					@extend .fa-university;}
+	.icon-sakai-delete{					@extend .fa-times;}
+	.icon-sakai-pdf{					@extend .fa-file-pdf-o;}
+	.icon-sakai-clip{					@extend .fa-paperclip;}
+	.icon-sakai-check{					@extend .fa-check;}
+	.icon-sakai-rsf-evaluation{				@extend .fa-commenting;}
+	.icon-sakai-bbb{					@extend .fa-video-camera;}
 }
```

Comment:
Sorry to be a pain here. Could you redo the original commit and just force push to your repo? So there's just one change on one line? I know it seems a bit pedantic, but I get it all the time, so it's the 'done thing'.